doc/error-handling.md

```diff
@@ -80,21 +80,21 @@ The following error levels are supported.
 * FATAL 
 
 ### MODERATE
-A recoverable error. Leads to an error log entry (LogErr) and continues execution.
+A recoverable error. Leads to an error log entry (*LogErr*) and continues execution.
```

Comment:
Uhh, the code constructs were supposed to be written with code formatting right ? So surrounded with \`\`.
e.g. `LogErr`

Reply:
I can do so.

Reply:
Done